File: Latihan3/Latihan3_CPP/Processor.cpp
#include <string>

using namespace std;

class Processor
{
private:
	/* data */
	// class attribute
	string name;
	int price;

public:
	Processor(/* args */){};

	Processor(string name, int price)
	{
		this->name = name;
		this->price = price;
	}

	// getter and setter
	void setName(string name)
	{
		this->name = name;
	}

	string getName()
	{
		return this->name;
	}

	void setPrice(int price)
	{
		this->price = price;
	}

	int getPrice()
	{
		return this->price;
	}

	// destructors
	~Processor(){};
};

File: Latihan3/Latihan3_PHP/Processor.php
<?php

class Processor
{
	// class attribute
	private $name;
	private $price;

	// constructors
	public function Processor($name = "", $price = 0)
	{
		$this->name = $name;
		$this->price = $price;
	}

	// getter and setter
	public function setName($name)
	{
		$this->name = $name;
	}
	public function getName()
	{
		return $this->name;
	}

	public function setPrice($price)
	{
		$this->price = $price;
	}

	public function getPrice()
	{
		return $this->price;
	}

	// destructors
	function __destruct()
	{
	}
}
